index.js

```diff
@@ -62,24 +62,16 @@ module.exports = function (options) {
 			if (errorList.length > 0) {
 				file.jscs.success = false;
 				file.jscs.errorCount = errorList.length;
-				file.jscs.errors = errorList;
+				file.jscs.errors = errors;
 			}
-
-			errorList.forEach(function (err) {
-				out.push(errors.explainError(err, true));
-			});
 		} catch (err) {
-			out.push(err.stack.replace('null:', file.relative + ':'));
+			console.error(err.stack.replace('null:', file.relative + ':'));
```

Comment:
@sindresorhus can you check if there's any input that would result in an error being thrown? I believe recent versions of JSCS report "Unexpected token" instead of throwing.
@markelog @mikesherov can you confirm whether my assumption is correct?